### ISSUE TITLE
Pointer declaration colouring seems weird

### ISSUE THREAD

**Issue by russel** [Emacs-D-Mode-Maintainers/Emacs-D-Mode#82]
It appears that:
```d
SomeType* v;
```
gets coloured differently to:
```d
SomeType * v;
```
and that the former is coloured correctly and the latter is not coloured at all.

**Comment by CyberShadow** [comment]
Both look fine here:

![](https://dump.thecybershadow.net/08ab32f49dfcc15e7b91a2fff402e591/11%3A35%3A31-upload.png)

Is there anything else in the surrounding code that would affect fontification?

**Comment by CyberShadow** [comment]
Managed to reproduce it (or a similar bug?) with:
```d
struct S
{
	this(Object o) {}
	this(Object* o) {}
}
```
`Object` in the second constructor definition isn't fontified as a type.

**Comment by mathias-lang-sociomantic** [comment]
Works for me (although fontification happens at different stages).

**Comment by nandryshak** [comment]
Changing the second constructor in CyberShadow's comment to:

    this(Object * o) {}

Break the fontification for me.

**Comment by CyberShadow** [comment]
This and some other issues should have been auto-closed when #93 was merged, but for whatever reason, they were not. I'm manually closing them now.
